src/main/resources/com/mathworks/ci/MatlabInstallation/config.jelly

```diff
@@ -0,0 +1,9 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    <f:entry title="Name: " field="name">
+        <f:textbox />
+    </f:entry>
+    <f:entry title="Matlab Root: " field="home">
```

Comment:
Is this a UI label? If so, I think MATLAB should be capitalized and the label should be in sentence case:

Matlab Root -> MATLAB root

Reply:
Done